### PR TITLE
fix: RN 73 compatibility

### PR DESCRIPTION
**Pull request by felipecsl**
https://github.com/react-native-community/discussions-and-proposals/issues/671

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -26,16 +26,20 @@ def getExtOrIntegerDefault(name) {
...
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault('minSdkVersion')
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
     versionCode 1
     versionName "1.0"
-    
+
   }
-  
+
```
Comment (edodusi) [inline android/build.gradle:42]: can you please remove these whitespaces?

Comment (felipecsl) [reply android/build.gradle:41]: there are no whitespaces here, just a line break

Comment (edodusi) [reply android/build.gradle:41]: yep, can you remove this change please?

File: android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.sparkfabrikreactnativeidfaaaid">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (edodusi) [inline android/src/main/AndroidManifest.xml:1]: why removing the package attribute?

File: android/build.gradle (hunk 1 above)
Comment (edodusi) [reply android/build.gradle:41]: I just want the commit to contain only the relevant logic (which is ok)

**Comment by edodusi** [comment]
closing this as the changes are merged, thank you @felipecsl

**Pull request closed without merging**